cron/worker/main.go

```diff
@@ -127,7 +127,7 @@ func createNetClients(ctx context.Context) (
 func startMetricsExporter() (monitoring.Exporter, error) {
 	exporter, err := monitoring.GetExporter()
 	if err != nil {
-		return nil, fmt.Errorf("error during NewStackDriverExporter: %w", err)
+		return nil, fmt.Errorf("error during monitoring.GetExporter: %w", err)
```

Comment:
shall we start using the new command to create errors?

Reply:
Yes, we should. Forgot about it in this PR.